File: webflux/simple-crud/src/main/java/br/com/leonardoferreira/poc/domain/response/CustomerResponse.java
package br.com.leonardoferreira.poc.domain.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CustomerResponse {

    private String id;

    private String name;

    private String createdAt;

    private String updatedAt;

}

File: jpa/jpa-one-to-one/src/main/kotlin/br/com/leonardoferreira/poc/domain/response/CustomerResponse.kt
package br.com.leonardoferreira.poc.domain.response

import com.fasterxml.jackson.annotation.JsonFormat
import java.time.LocalDateTime

data class CustomerResponse(

    val id: Long,

    val name: String,

    val email: String,

    val address: CustomerAddressResponse,

    @JsonFormat(pattern = "dd/MM/yyyy HH:mm:ss")
    val createdAt: LocalDateTime,

    @JsonFormat(pattern = "dd/MM/yyyy HH:mm:ss")
    val updatedAt: LocalDateTime

)